package.json

```diff
@@ -9,7 +9,7 @@
     "build:laboratory": "cd app/resonant-laboratory && webpack",
     "build:laboratory:tests": "webpack --config ./app/resonant-laboratory/webpack-testing.config.js",
     "test:laboratory:unit": "npm run test:laboratory:unit:raw | tap-spec",
-    "test:laboratory:unit:raw": "node build/resonant-laboratory/tests.js",
+    "test:laboratory:unit:raw": "for t in `ls build/resonant-laboratory/*.tests.js`; do node $t; done",
```

Comment:
I added a few of my own tests; if there's a better way to do this, I'm happy to change it.